module/src/components/calendarInput/calendarInput.component.tsx

```diff
@@ -107,6 +107,12 @@ export interface ICalendarInputProps<TValue extends Dates.DateLike>
 
   /** The character to show between the inputs, defaults to ":" */
   betweenInputs?: React.ReactNode;
+
+  /**
+   * Allows you to toggle the highlighting of todays date on the calendar
+   * - Defaults to true
+   */
+  highlightToday?: boolean;
```

Comment:
I reckon ICalendarInputProps should extend `Pick<ICalendarDisplayProps, 'highlightToday'>` to make it a bit safer if someone changes this

Reply:
also did you add it to CalendarView?